dashboard/app/models/ability.rb

```diff
@@ -107,9 +107,7 @@ def initialize(user)
         end
         can [:read, :start, :end, :workshop_survey_report, :summary, :filter], Pd::Workshop, facilitators: {id: user.id}
         can [:read, :update], Pd::Workshop, organizer_id: user.id
-        can :create, Pd::Workshop do |_|
-          Pd::CourseFacilitator.exists?(facilitator: user, course: Pd::Workshop::COURSE_CSF)
-        end
+        can :create, Pd::Workshop if Pd::CourseFacilitator.exists?(facilitator: user, course: Pd::Workshop::COURSE_CSF)
```

Comment:
@mehalshah, this is redundant with your work in https://github.com/code-dot-org/code-dot-org/pull/20963, and can be replaced by that. I added this when I discovered the issue yesterday, and added a [test](https://github.com/code-dot-org/code-dot-org/pull/20973/files#diff-6b83de5bff687d68cbe6468827b18b01R366)